src/main/java/picocli/CommandLine.java

```diff
@@ -3746,22 +3746,12 @@ private static String format(String formatString, Object... params) {
          * </p>
          * <b>A note on boolean options</b>
          * <p>
-         * By default picocli does not expect boolean options (also called "flags" or "switches") to have a parameter.
-         * You can make a boolean option take a required parameter by annotating your field with {@code arity="1"}.
+         * By default picocli allows boolean options (also called "flags" or "switches") to have an optional parameter.
+         * To remove the optional parameter, define the field with {@code arity = "0"}.
          * For example: </p>
-         * <pre>&#064;Option(names = "-v", arity = "1") boolean verbose;</pre>
-         * <p>
-         * Because this boolean field is defined with arity 1, the user must specify either {@code <program> -v false}
-         * or {@code <program> -v true}
-         * on the command line, or a {@link MissingParameterException} is thrown by the {@link #parse(String...)}
-         * method.
-         * </p><p>
-         * To make the boolean parameter possible but optional, define the field with {@code arity = "0..1"}.
-         * For example: </p>
-         * <pre>&#064;Option(names="-v", arity="0..1") boolean verbose;</pre>
-         * <p>This will accept any of the below without throwing an exception:</p>
```

Comment:
I would like to keep this example. Let's have an example for both arity=1 and arity=0.